app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -92,6 +92,30 @@ def diagnostic_codes
     disabilities.map { |disability| disability['diagnosticCode'] }
   end
 
+  def prepare_for_evss!
+    return unless forward_to_mas?
+
+    save_metadata(forward_to_mas: true)
+    insert_classification_codes
+  end
+
+  def send_post_evss_notifications!
+    send_rrd_completed_notification if rrd_job_selector.rrd_applicable?
+    notify_mas if form.dig('rrd_metadata', 'forward_to_mas')
```

Comment:
Wondering if we should have a `read_metadata('forward_to_mas')` (which calls `form.dig('rrd_metadata', 'forward_to_mas')`)  to complement `save_metadata(forward_to_mas: true)`.

Reply:
Yep, that's a great idea. I'll try to do it in another PR that doesn't already contain a boatload of refactoring.